lib/schedule/gtfs/direction.ex

```diff
@@ -46,4 +46,16 @@ defmodule Schedule.Gtfs.Direction do
   @spec id_from_string(String.t()) :: id()
   def id_from_string("0"), do: 0
   def id_from_string("1"), do: 1
+
+  @doc """
+  Returns a inbound / outbound terms from a Direction ID
+
+  iex> Schedule.Gtfs.Direction.id_from_string(0)
+  "Outbound"
+  iex> Schedule.Gtfs.Direction.id_from_string(1)
+  "Inbound"
```

Comment:
You might want to add a brief `moduledoc` to the top of this file, since right now it's tagged as `@moduledoc false`, thus not generating these docs.

Also, in order to get these examples to render nicely, you need to indent them 4 spaces.

```suggestion
      iex> Schedule.Gtfs.Direction.id_from_string(0)
      "Outbound"
      iex> Schedule.Gtfs.Direction.id_from_string(1)
      "Inbound"
```

# Before

<img width="874" alt="Screenshot 2024-08-28 at 12 10 00 PM" src="https://github.com/user-attachments/assets/9fc3db4e-692e-4fdb-a4d7-78787dd4b66a">

# After

<img width="874" alt="Screenshot 2024-08-28 at 12 09 27 PM" src="https://github.com/user-attachments/assets/e92af8ff-a79c-4e65-a4f0-a31a68535b49">


You can see the docs rendered by running `mix docs` and then bringing up `/path/to/skate/doc/api-reference.html` in the browser, or for this particular doc, `/path/to/skate/doc/Schedule.Gtfs.Direction.html`.

Reply:
I reverted the changes to this file anyway, given [this rec](https://github.com/mbta/skate/pull/2757#discussion_r1734968669)